internal/pkg/levee/levee.go

```diff
@@ -122,3 +203,25 @@ func getArgumentPropagator(c *config.Config, call *ssa.Call) ssa.Node {
 
 	return nil
 }
+
+// callValues collects the ssa.Values that are arguments to an ssa.Call.
+// in particular, it handles variadic arguments to some extent.
+// We don't want to analyze what the variadic argument points to (a slice), but we do
+// want to analyze the values *within* the slice.
+func callValues(c *ssa.Call) (values []ssa.Value) {
```

Comment:
I think this logic should remain close to other pointer-analysis-related functions, since it is very closely related to it and isn't likely to be used anywhere else.

In fact I think we may want to rename the `utils` package to something more evocative, or even remove it completely. Inspired by [this post](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common).